pkg/server/server_test.go

```diff
@@ -41,6 +42,7 @@ type serverTestEnv struct {
 	adminSigner    crypto.Signer
 	client         *mock.Client
 	certCol        *certificateauthority.CACertCollection
+	tlsCertCol     *certificateauthority.CACertCollection
```

Comment:
rename them such that it is clear which is for client-comm and which is for cluster-comm

Reply:
One is for signing, second is for sdk-server tls comm, none is for cluster comm